### ISSUE TITLE
Update PowerShell global tool to v7.0.2

### ISSUE THREAD

**Issue by kapsiR** [PowerShell/PowerShell#12964]
Since PowerShell [v7.0.2](https://github.com/PowerShell/PowerShell/releases/tag/v7.0.2) has been released several days ago, when will the [global tool](https://www.nuget.org/packages/PowerShell/) be updated?

**Comment by kapsiR** [comment]
[v7.0.2](https://www.nuget.org/packages/PowerShell/7.0.2) is online now, thanks! 👍

**Comment by adityapatwardhan** [comment]
Just released this morning. Thanks for filing the issue!

**Comment by tudor07** [comment]
@kapsiR How to update powershell?
I tried running `dotnet tool install --global PowerShell --version 7.0.2` but when I restart Terminal I still see powershell 7.0.1

**Comment by kapsiR** [comment]
@tudor07 When you want to update to the latest stable version, use `dotnet tool update -g powershell`.

**Important: Don't execute this within PowerShell, it can't update itself. (Close all instances of PowerShell and use cmd instead)**

**Comment by tudor07** [comment]
@kapsiR Still not working.
1. I closed all instances of PowerShell
2. Launch cmd
3. Run `dotnet tool update -g powershell` in cmd
4. Open Windows Terminal
5. This is what I see:
```
PowerShell 7.0.1

...

 A new PowerShell stable release is available: v7.0.2
```

**Comment by dmpe** [comment]
> @kapsiR Still not working.
> 
> 1. I closed all instances of PowerShell
> 2. Launch cmd
> 3. Run `dotnet tool update -g powershell` in cmd
> 4. Open Windows Terminal
> 5. This is what I see:
> 
> ```
> PowerShell 7.0.1
> 
> ...
> 
>  A new PowerShell stable release is available: v7.0.2
> ```
> 
> EDIT: Installed `.msi` package and it worked

can confirm that this is still the case. Probably deserves a new ticket...

dotnet 5 + centos 7 + installed latest 7.1.1. and it shows only as pwsh 7.0.3
